server/lib/chat/chat-models.ts

```diff
@@ -116,6 +117,41 @@ export async function getUserChannelEntriesForUser(
   }
 }
 
+export async function countUserJoinedChannels(
+  userId: SbUserId,
+  withClient?: DbClient,
+): Promise<number> {
+  const { client, done } = await db(withClient)
+  try {
+    const result = await client.query<DbUserChannelEntry>(sql`
+      SELECT *
+      FROM channel_users
+      WHERE user_id = ${userId};
```

Comment:
With whatever new additions to queries you write, I think that `official` channels should not count towards this limit